projects/components/src/header/page/page-header.component.ts

```diff
@@ -1,4 +1,4 @@
-import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
+import { ChangeDetectionStrategy, Component, ContentChild, Input, OnInit } from '@angular/core';
```

Comment:
Separated code paths for easier reading. If the FF for page time range is enabled, use a slightly different layout, lines 29-66. Otherwise, use old template - lines 68-87.  

For when the page TR FF is enabled, the template relies on directive content projection, regular projection is not caught - primary being first row of the header, secondary being second row of the header.

If TR FF is enabled but NavigationRedesign is **not** enabled, it displays normal, global time-range component.

If TR and Navigation Redesign are enabled, it displays the time range with page level behaviour (`ht-user-specified-time-range-selector`)